setup.py

```diff
@@ -56,7 +56,9 @@ def get_version():
         'pillow>=8.3',
         'pillow-jpls>=1.0',
         'pylibjpeg>=1.4',
-        'pylibjpeg-libjpeg>=1.3',
         'pylibjpeg-openjpeg>=1.2',
     ],
+    extras_requires={
+        'libjpeg': ['pylibjpeg-libjpeg>=1.3'],
```

Comment:
Shall we call this just `jpeg` rather than `libjpeg`?

Reply:
I think I prefer it as it is. If `[jpeg]` is an optional dependency, it makes it seem like without the optional dependency, no JPEG can be encoded or decoded, which is not true